monkestation/_maps/random_rooms.dm

```diff
@@ -0,0 +1,76 @@
+/datum/map_template/random_room/skwh_rdm158
+	name = "Large Abandoned Virology"
+	room_id = "skwh_rdm158_abandonedviro"
+	mappath = "monkestation/_maps/RandomRooms/10x10/skwh_rdm158_abandonedviro.dmm"
+	centerspawner = FALSE
+	template_height = 10
+	template_width = 10
+	weight = 2
+	stock = 3
```

Comment:
To me this feels like a LOT of cultures, I dont know what the non-random diseases do off the top of my head but they sound.. fairly bad. Also, the room does NOT need a chem dispenser, viro doesn't even get one.